internal/services/formatters/csharp/scs/formatter.go

```diff
@@ -59,20 +71,17 @@ func (f *Formatter) StartAnalysis(projectSubPath string) {
 	f.LogDebugWithReplace(messages.MsgDebugToolFinishAnalysis, tools.SecurityCodeScan, languages.CSharp)
 }
 
-//nolint:funlen
 func (f *Formatter) startSecurityCodeScan(projectSubPath string) (string, error) {
 	f.LogDebugWithReplace(messages.MsgDebugToolStartAnalysis, tools.SecurityCodeScan, languages.CSharp)
 
 	analysisData := f.getDockerConfig(projectSubPath)
-	if err := f.verifyIsSolutionError(analysisData.CMD); err != nil {
-		return analysisData.CMD, err
-	}
```

Comment:
I move this validation to `CheckOutputErrors` since the CMD is a hardcoded string command this function will always return nil and the solution error is from scs output.